transports/msmq/delayed-delivery_content_msmqtransport_[2,).partial.md

```diff
@@ -1,26 +1,33 @@
-Because MSMQ lacks a mechanism for sending delayed messages, the MSMQ transport uses an external store for delayed messages. Messages that are meant to be delivered later (e.g. [saga timeouts](/nservicebus/sagas/timeouts.md) or [delayed retries](/nservicebus/recoverability/configure-delayed-retries.md)) are persisted in the delayed message store until these are due. At that point the due messages are fetched from the store and dispatched to their ultimate destinations.
+Because MSMQ lacks a mechanism for sending delayed messages, the MSMQ transport uses an external store for delayed messages. Messages that are to be delivered later (e.g. [saga timeouts](/nservicebus/sagas/timeouts.md) or [delayed retries](/nservicebus/recoverability/configure-delayed-retries.md)) are persisted in the delayed message store until they are due. When a message is due, it is retreived from the store and dispatched to its destination.
 
-To configure the MSMQ transport to enable delayed message delivery use the following API:
+The MSMQ transport requires explicit configuration to enable delayed message delivery. For example:
 
 snippet: delayed-delivery
 
-The code above sets up a SQL Server delayed message store. The SQL Server store is the only store that ships with the MSMQ transport. By default the SQL Server store uses a table named after then endpoint with `.delayed` suffix but that table name can be customized. 
+The SQL Server delayed message store (`SqlServerDelayedMessageStore`) is the only delayed message store that ships with the MSMQ transport.
```

Comment:
I'm okay with the rephrasing this a bit, but I'd like to continue specifying that this table uses the delayed suffix and the name of the table can be customized.

Reply:
It's already in the code though. Is it worth repeating?

![image](https://user-images.githubusercontent.com/677704/130420645-37d80c76-6efd-4d94-a281-481b10ce4707.png)